Makefile

```
@@ -1,2 +1,7 @@
 build:
-	cd src/ios; carthage update --use-xcframeworks --platform ios; carthage build --use-xcframeworks --no-use-binaries --platform ios; rm -r Carthage/Build/SentrySwiftUI.xcframework
+	cd src/ios && \
+	rm -rf Carthage/Build &&
+  carthage update --use-xcframeworks --platform ios --no-use-binaries --cache-builds && \
+  carthage build --use-xcframeworks --no-use-binaries --platform iOS \
+      --xcodebuild-arguments='EXCLUDED_ARCHS=arm64e ONLY_ACTIVE_ARCH=NO' && \
+  rm -r Carthage/Build/SentrySwiftUI.xcframework
```

This works offline and should work online.

```suggestion
	cd src/ios && \
  carthage update --use-xcframeworks --platform ios && \
  EXCLUDED_ARCHS=arm64e carthage build --use-xcframeworks --no-use-binaries --platform ios && \
  rm -r Carthage/Build/SentrySwiftUI.xcframework
```